src/components/value-chart/AnimatedChart.js

```diff
@@ -0,0 +1,133 @@
+import { interpolatePath } from 'd3-interpolate-path';
+import * as shape from 'd3-shape';
+import { maxBy, minBy } from 'lodash';
+import React, { Component } from 'react';
+import { Animated, View } from 'react-native';
+import { parsePath } from 'react-native-redash';
+import Svg, { Path } from 'react-native-svg';
+import { deviceUtils } from '../../utils';
+
+const padding = 20;
+const additionalChartPadding = 999999;
```

Comment:
@wojtus7 mind explaining the reasoning behind this value?

is there any risk of this causing performance issues?

Reply:
I added this value to make flat lines on left and right side of the chart (those additional padding) 

Since the coming data is given in milliseconds and charts can show more than a year's timeframe I needed this biiiig value to make sure the chart will end beyond the screen boundaries. 
Also calculating it just on the edge of the screen on demand will create bend (depending on chart data), and sending those points far away will flatten the line without any need for different interpolators, and additional calculations.
I don't think this value has any impact on performance in current form.